.gitignore

```
@@ -1,26 +1,44 @@
-*.pyc
-*~
+# Byte-compiled / optimized / DLL files
+*.py[cd]
+
+# C extensions
 *.so
-*.pyd
-.*.swp
+
+# Distribution / packaging
+build/
+dist/
+*.egg
 *.egg-info
```

Running `python setup.py sdist bdist_wheel` generates the `.eggs` directory, which should be ignored.
```suggestion
*.egg-info
.eggs/
```